Library/Formula/gpac.rb

```diff
@@ -36,9 +36,11 @@ class Gpac < Formula
   depends_on "openjpeg" => :optional
 
   def install
-    args = ["--disable-wx",
+    # If pulseaudio is linked, the script detects it, but fails to link -lpulse.
+    args = ["--disable-pulseaudio",
+            "--disable-wx",
             "--prefix=#{prefix}",
-            "--mandir=#{man}"]
+            "--mandir=#{man}",]
```

Comment:
Note that I added this because,
```
Put a comma after the last item of a multiline array.
1 file inspected, 1 offense detected
```
is now raised with  `brew audit --strict` without this trailing comma. Introduced with ec3ead0a.

So, remove comma and push, even though it fails audit? Or roll back ec3ead0a? Or...?